upb/structdefs.int.h

```diff
@@ -114,6 +114,11 @@ struct upb_msgdef {
    * descriptor.upb.c. */
   bool map_entry;
 
+  /* Do primitive values in this message have explicit presence or not?
+   * TODO: set this flag properly for static descriptors; regenerate
```

Comment:
No, it hasn't been added to the code generator yet. The initialization macro always sets the flag to true right now.